src/api/app/views/webui/kiwi/_tabs.html.erb

```diff
@@ -10,17 +10,10 @@
 
 <div class="box-header header-tabs" id="package_tabs">
   <ul>
-    <%= tab 'kiwi', 'Kiwi', controller: 'kiwi/images', action: 'show', selected: true %>
+    <%= tab 'kiwi', 'Image Editor', controller: 'kiwi/images', action: 'show', selected: true %>
     <% if @image.package.name == "patchinfo" %>
```

Comment:
Do we need the patchinfo tab at all?

Reply:
I guess so... is a C&P from the package tabs...

Reply:
Then drop it (don't think it makes sense in this context) or move it at least to the Advanced section as well!

Reply:
Moved to Advanced